src/solvers/dgmulti/dg_parabolic.jl

```diff
@@ -60,7 +60,7 @@ end
 # TODO: can we avoid copying data?
 function transform_variables!(u_transformed, u, mesh,
                               equations_parabolic::AbstractEquationsParabolic,
-                              dg::DGMulti, parabolic_scheme, cache)
```

Comment:
Is this change directly related to LDG? If not, could we move it to a separate PR?

Reply:
Hm, so it is not strictly necessary. When preparing this, I searched every spot where `parabolic_scheme` occurrs and checked if it does something. Since this is not the case here, I decided to remove this.

But yeah, we could move this into a separate PR :+1:

Reply:
I see. On second thought, it seems fine to bundle it into this PR if it does not break any tests. No need to move it.